recipes/result/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (C) 2015-2020 Danilo Bargen and contributors
```

Comment:
This file is part of the pypi package. No need to add it here.

Reply:
Great, wasn't aware of this. Removed it!